stable/cluster-autoscaler/templates/priority-expander-configmap.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if hasKey .Values.extraArgs "expander" }}
+{{- if and (.Values.expanderPriorities) (eq .Values.extraArgs.expander "priority") -}}
```

Comment:
Is this valid config? If not, we should prevent it.
```
extraArgs:
  expander: priority
expanderPriorities: {}
```

Reply:
Yes. Earlier we thought to set default priorities to `expanderPriorities` but if users may already have taken steps outside the helm chart to get a configmap in place then this might break functionality for them. So we should not override the configmap unless the user explicitly sets `expanderPriorities` with the priorities.

Reply:
I'm not seeing this ConfigMap referenced anywhere. Is there something missing from this MR?

If we want to support an existing ConfigMap for expander priorities, shouldn't we allow users to specify the name?

Reply:
No, we don't need to reference ConfigMap anywhere. According to the priority expander configuration: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/expander/priority/readme.md#configuration
```
The ConfigMap must be named cluster-autoscaler-priority-expander and it must be placed in the same namespace as cluster autoscaler pod
```

Regarding the second point: Before this MR, a workaround is, create ConfigMap before installing cluster autoscaler with priority expander as:
```
This ConfigMap has to be created before cluster autoscaler with priority expander can be started
```
So users, who already have "cluster-autoscaler-priority-expander" in place and installing cluster autoscaler with the below config:
```
extraArgs:
  expander: priority
```
To support this, one more check is required to make sure that user/s really want to define a ConfigMap `cluster-autoscaler-priority-expander` with `expanderPriorities` priorities.

Reply:
Thanks for the explanation. Makes sense to me.

Reply:
Thank you very much for the quick review.